src/main/java/io/stargate/sgv2/jsonapi/service/bridge/executor/SchemaCache.java

```diff
@@ -5,23 +5,27 @@
 import io.smallrye.mutiny.Uni;
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.inject.Inject;
-import java.time.Duration;
+import java.util.Optional;
 
 /** Caches the vector enabled status for all the namespace in schema */
 @ApplicationScoped
 public class SchemaCache {
 
   @Inject private QueryExecutor queryExecutor;
 
-  private final Cache<String, NamespaceCache> schemaCache =
-      Caffeine.newBuilder().expireAfterWrite(Duration.ofSeconds(600)).maximumSize(100).build();
+  private final Cache<CacheKey, NamespaceCache> schemaCache =
```

Comment:
So would `CacheKey` also work for "no tenant" case (empty `Optional`)? I assume this would be the case for most of existing testing, usage (only CNDB using/passing tenant ids)

Reply:
IT cases are all run without tenant in OSS.